app/views/posts/index.html.erb

```diff
@@ -1,9 +1,34 @@
-<h1>Here is a list of posts for a given user</h1>
+<div class='user_div'>
+  <div class='user_photo'>
+    <%= image_tag @user.photo %>
+  </div>
+  <div class='user_info'>
+    <p class='user_name'><%= @user.name %></p>
+    <p class='number_of_posts'>Number of posts: <%= @user.post_counter %></p>
+  </div>
+</div>
 
-<ul>
+<div class='posts'>
+  <% post_number = 1 %>
   <% @posts.each do |post| %>
-    <li>
-      <%= post.title %>
-    </li>
+    <div class='individual_post'>
+      <h2>Post #<%= post_number %></h2>
+      <h3><%= post.title %></h3>
+      <p>
+        <%= truncate(post.text, length: 130, omission: '...') %>
+        <% if post.text.length > 20 %>
+          <% post_number += 1 %>
+        <% end %>
+      </p>
+      <div class='comments'>
+        <p class='number_of_comments'> Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+      </div>
+      <div class='comments_div'>
+        <h3> Comments: </h3>
+        <% post.comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
```

Comment:
- [ ] Kindly make sure to use the created methods in the model (`most_recent_comments`), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.